src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsnonassociationsapi/service/PrisonOffenderEventListener.kt

```diff
@@ -22,6 +22,7 @@ class PrisonOffenderEventListener(
   @SqsListener("nonassociations", factory = "hmppsQueueContainerFactoryProxy")
   @WithSpan(value = "hmpps-non-associations-prisoner-event-queue", kind = SpanKind.SERVER)
   fun onPrisonOffenderEvent(requestJson: String) {
+    log.info("Received event: [$requestJson]")
```

Comment:
do we want this to remain given that there is talk on slack about our logging being too costly?

Reply:
Merges are not that frequent and without it had to look for exceptions to find issue